ui/src/app/pages/workspace/workspace-about.spec.tsx

```diff
@@ -90,6 +90,14 @@ describe('WorkspaceAbout', () => {
       .toBeTruthy();
   });
 
+  it('should disable the share button is workspace is adminLocked', async () => {
+    currentWorkspaceStore.getValue().adminLocked = true;
```

Comment:
I'm not sure whether this change would affect other tests, so I'd prefer one that updates `currentWorkspaceStore.next()`.  Could you add the negative case as well?

Reply:
Done